### ISSUE TITLE
Still getting the error #851 : nipap-passwd: error: unrecognized arguments: --username

### ISSUE THREAD

**Issue by AJNOURI** [SpriteLink/NIPAP#892]
**Environment:**

> Distributor ID:	Ubuntu
> Description:	Ubuntu 14.04.1 LTS
> Release:	14.04
> Codename:	trusty
> Linux nipap 3.13.0-75-generic # 119-Ubuntu SMP Mon Jan 11 13:49:34 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**version:**

> nipap-passwd --version
> nipap-passwd version 0.28.4

Followed both installtion docs:
[https://spritelink.github.io/NIPAP/docs/install-debian.html](https://spritelink.github.io/NIPAP/docs/install-debian.html)
[https://spritelink.github.io/NIPAP/docs/config-www.html](https://spritelink.github.io/NIPAP/docs/config-www.html)

Getting this error #851

> sudo nipap-passwd add --username  nipap-www --password passx --name "User account for the web UI" --trusted
> usage: nipap-passwd [-h] [-u USER] [-p PASSWORD] [-n NAME] [-t] [-r]
>                     [-f DB_FILE] [-c CONFIG] [--version]
>                     [{list, add, delete, test-user, latest-version, create-database, upgrade-database}]
> nipap-passwd: error: unrecognized arguments: --username  nipap-www
> 

> nipap-passwd 
> The specified configuration file ('/etc/nipap/nipap.conf') does not exist

**Comment by rfederici** [comment]
I had this too,

If you look at the error, it's because the documentation is telling you to say --username, but the CLI expects --user (the [-u USER] flag -- all-caps means that's the long flag)

**Comment by plajjan** [comment]
@rfederici all-caps means it is the argument value. --help will show it all
```
kll@lingloiVM ~/kod/NIPAP/nipap $ ./nipap-passwd --help
usage: nipap-passwd [-h] [-u USER] [-p PASSWORD] [-n NAME] [-t] [--no-trusted]
                    [-r] [--no-readonly] [-f DB_FILE] [-c CONFIG] [--version]
                    [{list, add, delete, modify, test-user, latest-version, create-database, upgrade-database}]

NIPAP User configuration

positional arguments:
  {list, add, delete, modify, test-user, latest-version, create-database, upgrade-database}
                        define an action to execute

optional arguments:
  -h, --help            show this help message and exit
  -u USER, --username USER
                        username
  -p PASSWORD, --password PASSWORD
                        set user's password to PASSWORD
  -n NAME, --name NAME  set user's name to NAME
  -t, --trusted         mark user as trusted
  --no-trusted          mark user as not trusted
  -r, --readonly        set user to read only
  --no-readonly         set user to read write
  -f DB_FILE, --file DB_FILE
                        database file [default: read from config]
  -c CONFIG, --config CONFIG
                        read configuration from CONFIG
                        [default:/etc/nipap/nipap.conf]
  --version             show program's version number and exit
kll@lingloiVM ~/kod/NIPAP/nipap $ ./nipap-passwd --help
```

The mismatch between documentation and reality is fixed with 0.29 (which is in testing repo now) so I'm closing this one.

To be clear. Documentation says --username and nipap-passwd v0.29.0 or newer takes --username whereas nipap-passwd earlier than v0.29.0 (like 0.28.4) takes --user
